source/prerequisites/sql-server.txt

```diff
@@ -10,4 +10,117 @@ SQL Server Migration Prerequisites
    :depth: 1
    :class: singlecol
 
-SQL Server.
\ No newline at end of file
+Connecting Relational Migrator to SQL Server requires Change Data 
+Capture (CDC) mode enabled. When you set up a new connection, 
+Relational Migrator automatically generates the required SQL code for 
+configuring SQL Server. How you configure SQL Server varies 
+depending on if you are running a Snapshot or a Continuous sync job.
+
+- Snapshot sync jobs migrate all the data and then stops.
+- Continuous sync job run a snapshot and then enter a CDC stage to 
+  continuously replicate data changes.
+
+About this Task
+---------------
+
+- This page covers the details of the SQL scripts automatically 
+  generated by Relational Migrator.
+
+- Relational Migrator automatically detects SQL Server CDC settings when 
+  connecting to your database and generates the appropriate SQL 
+  statements to enable CDC if required.
+
+- You can also check SQL Server CDC settings by examining the ``is_tracked_by_cdc`` 
+  column in the `sys.tables catalog view <https://learn.microsoft.com/en-us/sql/relational-databases/system-catalog-views/sys-tables-transact-sql?view=sql-server-ver16>`__. 
+  A value of ``1`` for ``is_tracked_by_cdc`` indicates the table 
+  is enabled for change data capture.
+
+- Relational Migrator does not create any indexes on SQL Server to 
+  facilitate sync jobs. The create index creation permission is not 
+  required.
+
+Before you Begin
+----------------
+
+To enable the CDC option on SQL Server:
+
+- You must have the server level ``sysadmin`` role. 
+- You must have the database level ``db_owner`` role.
+- The `SQL Server agent <https://learn.microsoft.com/en-us/sql/ssms/agent/sql-server-agent?view=sql-server-ver16>`__
+  must be running.
+- The service account used to connect to SQL Server requires 
+  Select permission against all required tables.
+
+Steps
+-----
+
+The easiest way to set up your database is to run the automatically 
```

Comment:
[Suggestion]

In this case could we say "recommended" instead of "easiest" or simply omit that term/phrase? 

Maybe something like "To set up your database, run the script prompted by Relational Migrator during sync job creation."